app/uk/gov/hmrc/customs/rosmfrontend/controllers/auth/EnrolmentExtractor.scala

```diff
@@ -33,9 +34,17 @@ trait EnrolmentExtractor {
             .map(identifier => identifier.value)
       )
 
-  def enrolledEori(loggedInUser: LoggedInUserWithEnrolments): Option[Eori] =
+  def enrolledForService(loggedInUser: LoggedInUserWithEnrolments, service: Service.Value): Option[Eori] = service match {
+    case Service.ATar => enrolledATar(loggedInUser)
```

Comment:
if `T` is a capital letter `r` need to be too.

It's for method name and `Service.ATar` comment

Reply:
Good spot.  We need to agree if we are going to keep this enum, your map or something else (config based).
But we can do that later when this is all merged and integrated.